Assets/HoloToolkit/Input/Scripts/Focus/IPointingSource.cs

```diff
@@ -11,13 +12,22 @@ namespace HoloToolkit.Unity.InputModule
     /// </summary>
     public interface IPointingSource
     {
-        Ray Ray { get; }
+        bool InteractionEnabled { get; }
 
         float? ExtentOverride { get; }
 
+        [Obsolete("Will be removed in a later version. For equivalent behavior have Rays return a RayStep array with a single element.")]
+        Ray Ray { get; }
+
+        RayStep[] Rays { get; }
+
         LayerMask[] PrioritizedLayerMasksOverride { get; }
 
-        void UpdatePointer();
```

Comment:
We should also put this back and mark as obsolete with the new usage in the message.